marimo/_ai/llm/_impl.py

```diff
@@ -58,6 +58,8 @@ class openai(ChatModel):
             If not provided, the API key will be retrieved
             from the OPENAI_API_KEY environment variable or the user's config.
         base_url: The base URL to use
+        stream: Whether to stream responses. Defaults to True.
```

Comment:
we decided to leave this out and just always opt into streaming https://github.com/marimo-team/marimo/pull/7222